### ISSUE TITLE
Module build failed: Error: spawn

### ISSUE THREAD

**Issue by robksawyer** [imagemin/optipng-bin#91]
I'm getting the following when running `webpack-dev-server --config webpack.config.js --progress` with `node v8.2.1`.

I'm also running on `OSX Sierra 10.12.6 (16G29)`

```
ERROR in ./src/images/mobx.png
Module build failed: Error: spawn /Users/me/Sites/mycode/node_modules/imagemin-optipng/node_modules/optipng-bin/vendor/optipng ENOENT
    at exports._errnoException (util.js:1024:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:192:19)
    at onErrorNT (internal/child_process.js:374:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
 @ ./node_modules/css-loader?{"sourceMap":true}!./node_modules/postcss-loader/lib?{"sourceMap":true}!./node_modules/resolve-url-loader!./node_modules/sass-loader/lib/loader.js?{"sourceMap":true}!./src/styles/main.scss 6:7542-7571
 @ ./src/styles/main.scss
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:3001 webpack/hot/dev-server react-hot-loader/patch webpack-dev-server/client?http://0.0.0.0:3001 webpack/hot/only-dev-server babel-polyfill whatwg-fetch ./src/index

```

**Comment by robksawyer** [comment]
I'm getting the following when I run `node node_modules/optipng-bin/lib/install.js `.

```
  ⚠ self signed certificate in certificate chain
  ⚠ optipng pre-build test failed
  ℹ compiling from source
  ✖ RequestError
    at ClientRequest.<anonymous> (/Users/me/Sites/myproject/node_modules/optipng-bin/node_modules/got/index.js:74:21)
    at Object.onceWrapper (events.js:316:30)
    at emitOne (events.js:115:13)
    at ClientRequest.emit (events.js:210:7)
    at TLSSocket.socketErrorListener (_http_client.js:401:9)
    at emitOne (events.js:115:13)
    at TLSSocket.emit (events.js:210:7)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by robksawyer** [comment]
I still haven't come to a conclusion on this, but I replaced my `node_modules` directory with one that was compiling correctly and it had the following related modules in it.

```json
"dependencies": {
    "image-webpack-loader": "^2.0.0",
    "webpack": "^3.5.5",
    "webpack-dev-server": "^2.7.1"
},
"engines": {
    "node": "8.0.x",
    "yarn": "0.27.5"
 },
```

**Comment by ianks** [comment]
I am getting this same issue.

**Comment by zshnr** [comment]
It appears this issue is fixed in https://sourceforge.net/p/optipng/bugs/68/

Version 0.7.7 of OPTIPNG. However, this wrapper downloads 0.7.6. 

Shall I create a PR to change the version and then we can test if it works?

**Comment by 1000ch** [comment]
Fixed on `v5.0.0`.
